lib/safe-pg-migrations/plugins/statement_insurer/remove_column_index.rb

```diff
@@ -11,7 +11,8 @@ def remove_column_with_composite_index(table, column)
         return unless existing_indexes.any?
 
         error_message = <<~ERROR
-          Cannot drop column #{column} from table #{table} because composite index: #{existing_indexes.map(&:name).join(', ')} is present
+          Cannot drop column #{column} from table #{table} because composite index: #{existing_indexes.map(&:name).join(', ')} is present.#{' '}
+          If this index is still required, ensure it is recreated before the column is dropped.
```

Comment:
... because composite index `<index_name>` that contains this column is present
If this index is still required, ensure it is recreated without the columns to drop before dropping the existing index.